README.md

```diff
@@ -238,6 +238,7 @@ cog.out(pretty_list)
 | [<img src=".github/github.png" align="top" width="20" />](https://motrix.app/) | `motrix` | <i>A full-featured download manager.</i> |
 | [<img src=".github/github.png" align="top" width="20" />](https://github.com/SoongNoonien/mpdevil) | `mpdevil` | <i>A simple music browser for MPD.</i> |
 | [<img src=".github/github.png" align="top" width="20" />](https://agam778.github.io/MS-Office-Electron/) | `ms-office-electron` | <i>A Microsoft Office Online Desktop Client made with Electron.</i> |
+| [<img src=".github/direct.png" align="top" width="20" />](https://mullvad.net/) | `mullvad vpn` | <i>VPN service for speed and security.</i> |
```

Comment:
Please update the README file using [cog](https://pypi.org/project/cogapp).

Reply:
There appears to be an issue with the latest version of deb-get with prettylist (and thus using cog) where about 2 dozen apps (including mullvad) are getting incorrect icons and no Website or Summary listed (I logged it as #379)

Reply:
@GHMusicalCoder The prettylist (and hence readme/cog) issue are because of the breakage in ppa support (#365) in the latest release and will be fixed when #366 is merged or @flexiondotorg otherwise fixes it and makes another release.
If I get time later I'll try and sort a PR to your branch with the README cogged ....